manifests/candlepin.pp

```diff
@@ -80,19 +80,14 @@
       client_cert => $client_cert,
       refreshonly => true,
       subscribe   => Exec['create-nss-db'],
-      notify      => Service['qpidd'],
+      #notify      => Service['qpidd'],
```

Comment:
I don't think, this is necessary. It's a false dependency that causes issues in a distributed setup.